### PR TITLE
WD-7498 - Remove horizontal scrolling from controllers

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Remove min widths from tables.
- Remove horizontal scrolling from controller tables.

## QA

- Go to the controllers page.
- Check that the controllers table looks good at all screen sizes

## Details

https://warthogs.atlassian.net/browse/WD-7498

## Screenshots

Screen Recording 2023-11-23 at 12 35 47 pm

### CODE REVIEW

File: src/components/ModelTableList/_model-table-list.scss [hunk 1]
```diff
@@ -27,8 +26,6 @@
 }
 
 .models {
-  @extend %row-hover;
```
Comment (huwshimi) [inline src/components/ModelTableList/_model-table-list.scss:30]: These hover states were left from when you could click anywhere in the row and should have been removed when we changed to only clicking on the names.

File: src/pages/ControllersIndex/ControllerOverview/_controller-overview.scss [hunk 2]
```diff
@@ -9,9 +10,18 @@
 
   &__chart {
     margin-right: 2rem;
+
+    @include mobile {
```
Comment (huwshimi) [inline src/pages/ControllersIndex/ControllerOverview/_controller-overview.scss:14]: I made a few other little responsive improvements to the page.

File: src/pages/ControllersIndex/ControllersIndex.tsx [hunk 3]
```diff
@@ -298,9 +307,21 @@ function Details() {
       </div>
       <ControllersOverview />
       <div className="l-controllers-table u-overflow--auto">
-        {rows.length > 0 && <MainTable headers={headers} rows={rows} />}
+        {rows.length > 0 && (
+          <>
+            <h5 className="u-hide--large">{Label.DEFAULT}</h5>
```
Comment (huwshimi) [inline src/pages/ControllersIndex/ControllersIndex.tsx:312]: The titles for the tables appear in the header for the first column so I've added these titles above the tables on small screens.

File: src/pages/ControllersIndex/ControllersIndex.tsx [hunk 4]
```diff
@@ -101,7 +112,11 @@ function Details() {
   }
 
   const headers: MainTableHeader[] = [
-    { content: "default", sortKey: "name" },
+    {
+      content: <>{Label.DEFAULT}</>,
+      heading: Label.NAME,
```
Comment (huwshimi) [inline src/pages/ControllersIndex/ControllersIndex.tsx:117]: This replaces the title when on small screens.

File: src/pages/ControllersIndex/_controllers.scss [hunk 5]
```diff
@@ -43,10 +70,4 @@
     // cell.
     display: block !important;
   }
-
-  .controllers--registered-tooltip {
```
Comment (huwshimi) [inline src/pages/ControllersIndex/_controllers.scss:47]: I've changed this to use a normal tooltip component so this is no longer needed.

File: src/scss/custom/_tables.scss [hunk 6]
```diff
@@ -1,13 +1,3 @@
-// Site-wide Vanilla table overrides
-table {
-  min-width: 900px;
```
Comment (huwshimi) [inline src/scss/custom/_tables.scss:3]: This is what made the tables scroll horizontally so I've removed it for all tables.

**Review by vladimir-cucu — APPROVED** [review]
Works as expected locally. :rocket: 

Also, thanks for the comments! It made it easier to understand why certain lines were removed.

**Pull request merged**